imgee/storage.py

```diff
@@ -29,17 +29,23 @@ def get_resized_image(img, size, is_thumbnail=False):
     """
     registry = imgee.registry
```

Comment:
So, an `init_app()` function in the `tasks.py` (where `TaskRegistry` exists), and initialize the registry there and assign it to let's say `app.registry`, and call it from `imgee/__init__.py` so that it can be used from anywhere?